### PR TITLE
60 feature new design Druzhytskii

### PR DESCRIPTION
**Pull request by AndriiDruzhytskyi**
For correct work, this pull request needs new server changes - https://github.com/Social-projects-Rivne/U4U_Server/pull/52.

**Please check all app features before approve!!!**

### CODE REVIEW

File: src/components/app/app.scss [hunk 1]
```diff
@@ -214,3 +187,18 @@ body {
...
+// ::-webkit-scrollbar-track {
+// 	background-color: rgba(0, 0, 0, 0.4); 
+// }
+// ::-webkit-scrollbar-thumb {
+// 	background: #fff;
+// }
+// ::-webkit-scrollbar-thumb:hover {
+// 	background-color: rgba(255, 255, 255, 0.4); 
+// }
+
```
Comment (Dmitry528) [inline src/components/app/app.scss]: delete comment code

Comment (AndriiDruzhytskyi) [reply src/components/app/app.scss]: FIxed.

**Comment by AndriiDruzhytskyi** [comment]
Fixed many bugs.
Added a new logo.

For correct work, this pull request needs new server changes, switch server to next branch - https://github.com/Social-projects-Rivne/U4U_Server/pull/52.

**Please check all app features before approve!!!**
P.S. Also dont forgets to do `git pull` if you already checkout on this branch.

**Review by andrii-kondratiuk — APPROVED** [review]
Tested on firefox. Working as intended.

**Review by Dmitry528 — COMMENTED** [review]
1) Rivne zoo is rejected, but i can see it in "Places near by".
2) Pull current dev, becouse we have a functional with icons under image on place page.
3) All users have relative wishList list, yesterday i merge this functional and he is on current dev on client and server side ( pull dev on server && client )
4) When i redirect from map to Rivne places i have this error " TypeError: location is undefined "

**Review by Dmitry528 — COMMENTED** [review]
Sorry, point #2 is on review

**Comment by AndriiDruzhytskyi** [comment]
> 1. Rivne zoo is rejected, but i can see it in "Places near by".
> 2. Pull current dev, becouse we have a functional with icons under image on place page.
> 3. All users have relative wishList list, yesterday i merge this functional and he is on current dev on client and server side ( pull dev on server && client )
> 4. When i redirect from map to Rivne places i have this error " TypeError: location is undefined "

1. Fixed.
2. Still waiting.
3. Current dev was merged.
4. Crash fixed in https://github.com/Social-projects-Rivne/U4U_Server/pull/52.

**Comment by AndriiDruzhytskyi** [comment]
Added new dependencies:
1._@fortawesome/free-regular-svg-icons": "^5.11.2_
2. _@types/react-router-dom": "^5.1.3_

Please **reinstall node modules** folder before starting to check this pull request.

For correct work, this pull request needs new server changes, switch server to next branch - https://github.com/Social-projects-Rivne/U4U_Server/pull/52.

**Check all app features before approve!!!**
P.S. Also dont forgets to do git pull if you already checkout on this branch.

**Review by andrii-kondratiuk — APPROVED** [review]
Tested on Google Chrome and Mozilla Firefox multiple times. Everything working fine.

**Review by Dmitry528 — APPROVED** [review]
Good work

**Review by marrybbb — APPROVED** [review]
great work!!!

**Pull request merged**